devops/actions/llvm_test_suite/action.yml

```diff
@@ -115,6 +102,6 @@ runs:
     if: always()
     run: |
       rm -rf toolchain
-      rm -rf lit
       rm -rf build
+      rm -rf build-e2e
       rm -rf llvm_test_suite
```

Comment:
I have doubts that "Cleanup" step is doing anything useful.
The clean-up job registered by the caller of this action will remove the whole workspace including all the directories listed here.
This is probably an idea for follow-up improvements.